third_party/dotnet/devtools/src/generator/CodeGen/Utility.cs

```diff
@@ -20,10 +21,13 @@ public static class Utility
         /// <returns></returns>
         public static string ReplaceTokensInPath(string path, string className, CodeGeneratorContext context, CodeGenerationSettings settings)
         {
+            string separator = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? "\\" : "/";
+
             path = path.Replace("{{className}}", className);
             path = path.Replace("{{rootNamespace}}", settings.RootNamespace);
             path = path.Replace("{{templatePath}}", settings.TemplatesPath);
             path = path.Replace("{{domainName}}", context.Domain.Name);
+            path = path.Replace("{{separator}}", separator);
```

Comment:
Or, just use `Path.DirectorySeparatorChar` here instead.

Reply:
Yes, this is the better way to do this; I found other examples in the code and copied them. Thanks.